app/src/main/java/com/vuongnq14798/vuongmp3/base/BaseActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.vuongnq14798.vuongmp3.base
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+abstract class BaseActivity : AppCompatActivity() {
+
+    abstract val layoutResId: Int
```

Comment:
thuộc tính này nên cho thêm annotation `@get:LayoutRes` để nó nhận định khi những thằng con override nó phải bắt buộc là một resource layout nếu không sẽ báo lỗi 
```suggestion
    @get:LayoutRes
    abstract val layoutResId: Int
```